traffic_ops/app/lib/API/Tenant.pm

```diff
@@ -138,21 +138,21 @@ sub update {
 			#Parent tenant to be set is not under user's tenancy
 			return $self->alert("Invalid parent tenant. This tenant is not available to you for parent assignment.");
 		}
-		my $parent_depth = $tenant_utils->get_tenant_heirarchy_depth($tenants_data, $params->{parentId});
+		my $parent_depth = $tenant_utils->get_tenant_hierarchy_depth($tenants_data, $params->{parentId});
```

Comment:
Did you check to make sure that these function name changes didn't break anything?

Reply:
The corresponding function definitions in the Utils Module were updated along with the call updates here in the API Module. (See traffic_ops/app/lib/Utils/Tenant.pm down below). Thanks for checking!

Reply:
Alright, well I've got no problem with it if nobody else does